### ISSUE TITLE
[Bug] SplitTooLongLines alternates indentation in variable section

### ISSUE THREAD

**Issue by robinmatz** [MarketSquare/robotframework-tidy#363]
##Description:
Given I have the following robot file
```robot
# file.robot

*** Variables ***
${LONG_VAR}    verylongvariable1234123412341234124123412341234123412341234123412341234123412341234123412341234123412341
${SHORT_VAR}    abc
```

When I run `robotidy file.robot`, it gives me the following output

```robot
*** Variables ***
${LONG_VAR}
...    verylongvariable1234123412341234124123412341234123412341234123412341234123412341234123412341234123412341
${SHORT_VAR}    abc
```

However, when I run `robotidy file.robot` again, it changes the output to

```robot
*** Variables ***
${LONG_VAR}
...             verylongvariable1234123412341234124123412341234123412341234123412341234123412341234123412341234123412341
${SHORT_VAR}    abc
```

##Expected behavior:
robotidy should format the code consistently, not changing the space count on every run.
Preferably, the output should look like after the second run, being in line with ${SHORT_VAR}

##Affected version:
3.0

**Comment by bhirsz** [comment]
It was probably fixed when I fixed #360 (sounds familiar). It's on the main branch already, I will try to create 3.0.1 or 3.1 release soon

**Comment by bhirsz** [comment]
But of course I will first try to reproduce it on the old code /current code and see if that fix helps.

**Comment by bhirsz** [comment]
I can confirm it does repeatably format the file in 3.0.0 but using the code from the main branch it does stop formatting the file after 2 re-runs (and it looks like in you last example). I will close this issue as fixed after the next release.

**Comment by bhirsz** [comment]
Released as 3.1.0 (https://github.com/MarketSquare/robotframework-tidy/releases/tag/3.1.0)
